src/adhocracy_core/adhocracy_core/sheets/test_description.py

```diff
@@ -0,0 +1,35 @@
+from pyramid import testing
+from pytest import fixture
+
+
+class TestDescriptionSheet:
+
+    @fixture
+    def meta(self):
+        from adhocracy_core.sheets.description import description_meta
+        return description_meta
+
+    def test_create(self, meta, context):
+        from adhocracy_core.sheets.description import IDescription
+        from adhocracy_core.sheets.description import DescriptionSchema
+        from adhocracy_core.sheets import AnnotationStorageSheet
+        inst = meta.sheet_class(meta, context)
+        assert isinstance(inst, AnnotationStorageSheet)
+        assert inst.meta.isheet == IDescription
+        assert inst.meta.schema_class == DescriptionSchema
+        assert inst.meta.editable is True
+        assert inst.meta.create_mandatory is False
+
+    def test_get_empty(self, meta, context):
+        inst = meta.sheet_class(meta, context)
+        assert inst.get() == {'description': ''}
+
+
+def test_includeme_register_description_sheet(config):
```

Comment:
Maybe we should move this test in the class, like here: https://github.com/liqd/adhocracy3.mercator/pull/1034/files#diff-81b4a36673cf425b0f8f52cc4daf697eR96